### DISCUSSION TITLE
bun.js size and memory usage

### DISCUSSION THREAD

**Discussion by vjpr** [oven-sh/bun#812] [General]
When thinking about using Bun as a desktop replacement over Electron (https://github.com/oven-sh/bun/discussions/790), I am wondering how it compares to Node.js in terms of download size and also memory usage.

# Download size

`bun` v0.1.4 = 37MB uncompressed / 12.5MB compressed
`node` v18.6.0 = 86.5MB uncompressed / 29.3MB compressed

Does bun have a production-only build that could be even slimmer?

# Memory usage

I guess we only need to look at JSC vs V8 for this.

**Comment by KaKi87** [comment]
Well, this doesn't look promising :sweat_smile: 
```bash
$ bun -v && du -h $(which bun)
1.0.26
91M	/home/kaki/.bun/bin/bun
$ node -v && du -h $(which node)
v20.5.1
92M	/usr/bin/node
```
